packages/instantsearch-ui-components/src/components/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './chat/ChatMessage';
 export * from './chat/ChatToggleButton';
 export * from './chat/ChatHeader';
 export * from './chat/types';
+export * from './autocomplete';
```

Comment:
not consistent between chat (no barrel file) and autocomplete (barrel file). I don't have a specific preference, but we'll want to consolidate after both sprints are over.